src/constants/options/filters.js

```diff
@@ -1,5 +1,5 @@
-export const POPULAR_FILTERS = ['TRENDING', 'NEW', 'PROMOTED'];
-export const POPULAR_FILTERS_VALUE = ['trending', 'created', 'promoted'];
+export const POPULAR_FILTERS = ['FEED', 'TOP', 'HOT', 'NEW'];
+export const POPULAR_FILTERS_VALUE = ['feed', 'trending', 'hot', 'created'];
```

Comment:
i was use these values for new url scheme feature, you forgot to add `promoted` or you removed that

Reply:
also we should handle promoted keyword on search and deep link right? @feruzm

Reply:
let's remove promoted @mistikk